beacon-chain/cache/depositsnapshot/finalized_node.go

```diff
@@ -0,0 +1,34 @@
+package depositsnapshot
+
+import "github.com/pkg/errors"
+
+// Finalized represents a finalized node and satisfies the MerkleTreeNode interface.
+type Finalized struct {
+	deposits uint64
+	hash     [32]byte
+}
+
+// GetRoot returns the root of the Merkle tree.
+func (f *Finalized) GetRoot() [32]byte {
+	return f.hash
+}
+
+// IsFull returns whether there is space left for deposits.
+func (f *Finalized) IsFull() bool {
+	return true
+}
```

Comment:
Same feedback to add commentary about why this is always full.

Reply:
Same as above.